packages/cozy-notifications/src/mjml.js

```diff
@@ -0,0 +1,15 @@
+import mjml2html from 'mjml'
```

Comment:
There is no need for our custom cozy-mjml?

Reply:
See https://github.com/cozy/cozy-libs/pull/807#discussion_r327967036